openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -46,6 +49,11 @@
     'hr_expense_analytic_plans': 'hr_expense_analytic_distribution',
     # OCA/project
     'project_work_time_control': 'project_timesheet_time_control',
+    # OCA/manufacture-reporting
+    'mrp_bom_structure_xls': 'mrp_bom_structure_xlsx',
+    'mrp_bom_structure_xls_level_1': 'mrp_bom_structure_xlsx_level_1',
+    # OCA/reporting-engine
+    'report_xls': 'report_xlsx',
```

Comment:
I disagree. `report_xls` of `reporting-engine` is not installable in v9. We are doing thinks based on what OCA has now in the repositories, not based on what some individuals may do.